src/models/device.ts

```diff
@@ -2984,31 +2997,25 @@ const getDeviceModel = function (
 			): Promise<void> {
 				value = String(value);
 
-				let deviceFilter;
-				if (isId(uuidOrId)) {
```

Comment:
It would be great if we could preserve this optimization, since I think the UI relies on it to make less requests.
Other than that LGTM :+1: